packages/ra-ui-materialui/src/layout/MenuItemLink.tsx

```diff
@@ -137,17 +137,16 @@ export const MenuItemLink = forwardRef((props: MenuItemLinkProps, ref) => {
     );
 });
 
-interface Props {
```

Comment:
This change is required to let TS compile the Menu component after it reexports the MenuItem component.